lib/biokbase/catalog/baseclient.py

```diff
@@ -121,7 +121,7 @@ def __init__(
             self, url=None, timeout=30 * 60, user_id=None,
             password=None, token=None, ignore_authrc=False,
             trust_all_ssl_certificates=False,
-            auth_svc='https://kbase.us/services/authorization/Sessions/Login',
+            auth_svc='https://ci.kbase.us/services/auth/api/legacy/KBase/Sessions/Login',
```

Comment:
This is generated code - did you regenerate or edit?

Reply:
Edited. Reverting this change.